doc/reference/kernel/synchronization/mutexes.rst

```diff
@@ -166,3 +166,25 @@ API Reference
 
 .. doxygengroup:: mutex_apis
    :project: Zephyr
+
+Futex API Reference
+*********************************
```

Comment:
Please check format - I think the underline (*) needs to be same length as the title line above it.